<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Demo</class>
 <widget class="QDialog" name="Demo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Демонстрация</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="demoInsertSortBtn">
         <property name="minimumSize">
          <size>
           <width>131</width>
           <height>41</height>
          </size>
         </property>
         <property name="text">
          <string>Сортировка вставками</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="demoMergeSortBtn">
         <property name="minimumSize">
          <size>
           <width>121</width>
           <height>41</height>
          </size>
         </property>
         <property name="text">
          <string>Сортировка слиянием</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="demoTimSortbtn">
         <property name="minimumSize">
          <size>
           <width>121</width>
           <height>41</height>
          </size>
         </property>
         <property name="text">
          <string>Сортировка Timsort </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="minimumSize">
          <size>
           <width>108</width>
           <height>41</height>
          </size>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
         <property name="placeholderText">
          <string>Вводите массив для сортировки здесь</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="randomBox">
         <property name="minimumSize">
          <size>
           <width>74</width>
           <height>41</height>
          </size>
         </property>
         <property name="whatsThis">
          <string>Случайным образом заполняет массив</string>
         </property>
         <property name="text">
          <string>Случайно</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sortBtn">
         <property name="minimumSize">
          <size>
           <width>91</width>
           <height>41</height>
          </size>
         </property>
         <property name="text">
          <string>Сортировать</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphicsView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
